source/gui/contextHelp.py

```diff
@@ -7,6 +7,7 @@
 import tempfile
 import typing
 
+import ui

```

Comment:
`ui` is imported lazily in other places in this file. Are you certain that importing it at the top is safe? If not then consider making this import dynamic.

Reply:
I believe this is safe as `gui/__init__.py` imports `ui`